mlrun/projects/pipelines.py

```diff
@@ -675,9 +677,7 @@ def get_state(run_id, project=None):
         return ""
 
     @staticmethod
-    def wait_for_completion(
-        run_id, project=None, timeout=None, expected_statuses=None, run_object=None
```

Comment:
same for `run_object` and how come is this function not implemented?

Reply:
`local` engine cannot support `wait_for_completion` so it is not implemented.

Reply:
why?

Reply:
because it is running locally the pipeline blocks the continuity of the running code, and by default the user has to wait for completion.